e2e/utils.ts

```
@@ -0,0 +1,43 @@
+export async function waitForStablePage(page: Page) {
+  await page.waitForLoadState('networkidle')
+  await page.evaluate(() => document.fonts.ready)
+
+  await page.evaluate(() => new Promise<void>((resolve) => {
+    const start = Date.now()
+
+    const waitForIdle = () => {
+      if ('requestIdleCallback' in window) {
+        requestIdleCallback(() => {
+          if (Date.now() - start >= 500)
+            resolve()
```

ESLint

[error] Expected { after 'if' condition.

(curly)